hacking/ticket_stubs/collections.md

```diff
@@ -2,13 +2,15 @@ Hi!
 
 Thank you very much for your submission to Ansible. It sincerely means a lot to us that you've taken the time to contribute.
 
-Per recent announcements at AnsibleFest Atlanta 2019, going forward we are no longer accepting new one off modules/plugins, or modules/plugins that create a new category.
+Since Ansible 2.10 we are no longer accepting new one off modules/plugins into Ansible core.  However, we recommend looking into providing this functionality through Ansible Galaxy via Ansible Collections. You can find more information about collections at:
```

Comment:
```suggestion
Since Ansible 2.10 we are no longer accepting new modules/plugins into Ansible core.  However, we recommend looking into providing this functionality through Ansible Galaxy via Ansible Collections. You can find more information about collections at:
```